docs/en/customising-vanilla.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Customising Vanilla
```

Comment:
Maybe `Importing parts of Vanilla`

Reply:
This section also includes "Overriding default settings" though

Reply:
Maybe "Customising include of Vanilla"

Reply:
@anthonydillon Nooo, that sounds weird. I think "Customising Vanilla" is already self-explanatory. @kwm14 agrees

Reply:
👍